public/locales/en/translation.json

```diff
@@ -37,6 +37,32 @@
     "largeFilesWarning": "If you have issues downloading files larger than 1GB, installing <1>IPFS Desktop</1> or <3>IPFS Companion</3> can improve performance.",
     "viewOnGateway": "View on IPFS"
   },
+  "modal": {
+    "how": {
+      "title": "How IPFS Share works",
+      "ariaLabel": "Modal explaining how IPFS Share works",
+      "copy": "IPFS Share is a simple, free way to share files without needing to rely on services run by big corporations. Instead, it uses the peer-to-peer IPFS protocol for sharing directly between devices. IPFS is built around content addressing — keeping track of files based on what they contain, not where they're located — which means that files can't be modified without changing their addresses too. In other words, when you download a file using IPFS, you can be sure that it's exactly the same as the sender intended.",
```

Comment:
You talk quite a bit about content addressing in this one. Would it be worth adding an additional link to the ProtoSchool lesson on Content Addressing here for the receivers as you did for the senders?

Reply:
Good idea. Fixed in https://github.com/ipfs-shipyard/ipfs-share-files/pull/104/commits/e706d30f760b6951d510730ef699669811e89c24